packages/st2mistral/debian/mistral-api.init

```diff
@@ -69,6 +70,18 @@ do_start()
   #   1 if daemon was already running
   #   2 if daemon could not be started
   lsb_running && return 1
+
+  # /run is a tmpfs mount, so we need to check the directory exists and create.
+  # Otherwise the service will not start following a reboot.
+  if [ ! -d "$RUN_DIR" ] ; then
+      if ! mkdir -p "$RUN_DIR" ; then
```

Comment:
looks. good. As of `mkdir -p` it's a highly rare circumstance that it can fail. Usually it never fails, i.e. exits with 0 status even when the directory exits. Can only happen if there's not enough permissions, since init scripts are handled with root privileges, so the failure should never happen.